test/image/mocks/gl3d_cone-newplot_reversed_ranges.json

```diff
@@ -1,27 +1 @@
-{
-  "data": [
-    {"x": [0.2], "y": [0.4], "z": [-0.6], "u": [1], "v": [1], "w": [0], "type": "cone"},
-    {"x": [0.4], "y": [-0.6], "z": [0.2], "u": [1], "v": [0], "w": [1], "type": "cone"},
-    {"x": [-0.6], "y": [0.2], "z": [0.4], "u": [0], "v": [1], "w": [1], "type": "cone"},
-    {"x": [-0.2], "y": [-0.4], "z": [0.6], "u": [-1], "v": [-1], "w": [0], "type": "cone"},
-    {"x": [-0.4], "y": [0.6], "z": [-0.2], "u": [-1], "v": [0], "w": [-1], "type": "cone"},
-    {"x": [0.6], "y": [-0.2], "z": [-0.4], "u": [0], "v": [-1], "w": [-1], "type": "cone"}
-  ],
-  "layout": {
-    "height": 758, 
-    "width": 1310, 
-    "title": "Cone objects with Y-axis using autorange: 'reversed'", 
-    "scene": {
-      "yaxis": {
-        "autorange": "reversed"
-      },
-      "camera": {
-          "eye": {
-              "x": 1,
-              "y": -1.5,
-              "z": 1.25
-          }
-      }    
-    }
-  }
-}
+{"data":[{"x":[0.2],"y":[0.4],"z":[-0.6],"u":[1],"v":[1],"w":[0],"type":"cone"},{"x":[0.4],"y":[-0.6],"z":[0.2],"u":[1],"v":[0],"w":[1],"type":"cone"},{"x":[-0.6],"y":[0.2],"z":[0.4],"u":[0],"v":[1],"w":[1],"type":"cone"},{"x":[-0.2],"y":[-0.4],"z":[0.6],"u":[-1],"v":[-1],"w":[0],"type":"cone"},{"x":[-0.4],"y":[0.6],"z":[-0.2],"u":[-1],"v":[0],"w":[-1],"type":"cone"},{"x":[0.6],"y":[-0.2],"z":[-0.4],"u":[0],"v":[-1],"w":[-1],"type":"cone"}],"layout":{"height":758,"width":1310,"title":"Cone objects with Y-axis using autorange: 'reversed'","scene":{"yaxis":{"autorange":"reversed"},"camera":{"eye":{"x":1,"y":-1.5,"z":1.25}}}}}
```

Comment:
Why?

Reply:
Those new mock are very nice though. Thanks very much!

Reply:
When try running image tests (on CircleCi) it showed syntax errors. I thought removing the white spaces may help pass the test. But I personally like the version which is human readable.  If prefer I could fix the json mocks?

Reply:
Hmm. The only syntax tests we run on JSON mocks are:

- [assert that their file name have no upper case](https://github.com/plotly/plotly.js/blob/5cc37e3504cfa4c1cf1bd632a97b66252126d99a/tasks/test_syntax.js#L224-L262)
- [assert that they don't contain trailing newlines](https://github.com/plotly/plotly.js/blob/5cc37e3504cfa4c1cf1bd632a97b66252126d99a/tasks/test_syntax.js#L264-L301)

So, I suspect your json files had trailing newlines. You can (and should) run those test locally with `npm run test-syntax`.

---------

Would you reformatting those files? Thanks!

Reply:
The new json mocks are reformatted.